src/miner.cpp

```diff
@@ -228,7 +229,52 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
         CBlockHeader header = (*itr).second->GetBlockHeader();
 
         uint32_t nTime = pblock->nTime - header.nTime;
-        coinbaseTx.vout[0].nValue = GetProofOfStakeReward(nHeight, tx->vout[pblock->vtx[1]->vin[0].prevout.n].nValue, nTime, chainparams.GetConsensus());
+        CAmount nBlockReward = GetProofOfStakeReward(nHeight, tx->vout[pblock->vtx[1]->vin[0].prevout.n].nValue, nTime, chainparams.GetConsensus());
+
+        // GetRewardPct tekina something
+        //kokokara
+        std::vector<std::pair<CTxDestination, int>>rewardPct;
+        rewardPct.resize(2);
+        CTxDestination address;
+        if(!ExtractDestination(scriptPubKey, address))
+        {
+            return nullptr;
+        }
+        rewardPct[0].first = address;
+        rewardPct[0].second = 40;
+        rewardPct[1].first = address;
+        rewardPct[1].second = 60;
+        //kokomade
+
+        std::vector<std::pair<CScript, CAmount>> rewardValue;
```

Comment:
Please make a new function that calculating reward value. (for making test code)